content/blog/2021-01-21-unpack-cfr.Rmd

```diff
@@ -0,0 +1,239 @@
+---
+title: "Unpacking the Drop in COVID-19 Case Fatality Rates"
+author: "Helen Zhou, Cheng Cheng, Jeremy C. Weiss, Zachary C. Lipton"
+output:
+  bookdown::html_document2: default
+bibliography: unpack-cfr/refs.bib
+link-citations: yes
```

Comment:
please add also the other header elements, see the other blog posts for example, including:

 * author list
 * acknowledgments
 * summary
 * hero image
 * tags
 * ...

see https://github.com/cmu-delphi/www-main/blob/8900c334d1b786ecf6ff04e2426241a727565f1d/content/blog/2021-01-15-causal-effect-mobility.Rmd#L1-L28